components/fs-data-provider/src/bin/icu4x-cldr-export.rs

```diff
@@ -0,0 +1,192 @@
+use clap::{App, Arg, ArgGroup};
```

Comment:
do you expect more binaries? If this is the only binary out of this app, you can also just do `src/main.rs` and it'll work with `cargo run` by default. `bin/*` is mostly useful when you have a library with side binaries. I'm not sure which one is that, so optional :)

Reply:
I kept it as a separate binary because I don't see this binary strictly tied to this component.  It's really a hybrid between cldr-json-data-provider and fs-data-provider.